### DISCUSSION TITLE
Avalon - Angular Upgrade + Standalone Components

### DISCUSSION THREAD

**Discussion by ZacharyPataky** [primefaces/community#3241] [PrimeNG Templates]
Hi team,

What is the timeline for upgrading the PrimeNG Avalon template to use standalone components?  My company wants to overhaul our various FE applications, and I'd rather hold off and perform one wave of rewrites if this feature is soon incoming.  Otherwise, I'll need to accommodate two waves: one for Avalon as it is, then another for standalone components.

**Comment by dizzymisslizzy916** [comment]
I'm also interested and would like an estimated delivery date. Thanks.

**Comment by buecherlstefan** [comment]
Any news?

**Reply by ZacharyPataky** [reply]
No dice

**Comment by AshishFreak** [comment]
Hi can you share Avalon theme with me ?
